models/user.py

```diff
@@ -20,7 +20,7 @@ class UserModel(BaseModel):
     __tablename__ = 'users'
 
     id = db.Column(db.Integer, primary_key=True)
-    email = db.Column(db.String(100))
+    email = db.Column(db.String(100), unique=True)
```

Comment:
should probably also add this too:
```suggestion
    email = db.Column(db.String(100), unique=True, nullable=False)
```